core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -175,20 +179,20 @@ class KafkaController(val config: KafkaConfig,
   /* single-thread scheduler to clean expired tokens */
   private val tokenCleanScheduler = new KafkaScheduler(1, true, "delegation-token-cleaner")
 
-  metricsGroup.newGauge(ZkMigrationStateMetricName, () => ZkMigrationState.ZK.value().intValue())
-  metricsGroup.newGauge(ActiveControllerCountMetricName, () => if (isActive) 1 else 0)
-  metricsGroup.newGauge(OfflinePartitionsCountMetricName, () => offlinePartitionCount)
-  metricsGroup.newGauge(PreferredReplicaImbalanceCountMetricName, () => preferredReplicaImbalanceCount)
-  metricsGroup.newGauge(ControllerStateMetricName, () => state.value)
-  metricsGroup.newGauge(GlobalTopicCountMetricName, () => globalTopicCount)
-  metricsGroup.newGauge(GlobalPartitionCountMetricName, () => globalPartitionCount)
-  metricsGroup.newGauge(TopicsToDeleteCountMetricName, () => topicsToDeleteCount)
-  metricsGroup.newGauge(ReplicasToDeleteCountMetricName, () => replicasToDeleteCount)
-  metricsGroup.newGauge(TopicsIneligibleToDeleteCountMetricName, () => ineligibleTopicsToDeleteCount)
-  metricsGroup.newGauge(ReplicasIneligibleToDeleteCountMetricName, () => ineligibleReplicasToDeleteCount)
-  metricsGroup.newGauge(ActiveBrokerCountMetricName, () => activeBrokerCount)
+  metricsGroup.newGauge(ZkMigrationStateMetricName, () => ZkMigrationState.ZK.value().intValue(), metricTags.asJava)
+  metricsGroup.newGauge(ActiveControllerCountMetricName, () => if (isActive) 1 else 0, metricTags.asJava)
+  metricsGroup.newGauge(OfflinePartitionsCountMetricName, () => offlinePartitionCount, metricTags.asJava)
+  metricsGroup.newGauge(PreferredReplicaImbalanceCountMetricName, () => preferredReplicaImbalanceCount, metricTags.asJava)
+  metricsGroup.newGauge(ControllerStateMetricName, () => state.value, metricTags.asJava)
+  metricsGroup.newGauge(GlobalTopicCountMetricName, () => globalTopicCount, metricTags.asJava)
+  metricsGroup.newGauge(GlobalPartitionCountMetricName, () => globalPartitionCount, metricTags.asJava)
+  metricsGroup.newGauge(TopicsToDeleteCountMetricName, () => topicsToDeleteCount, metricTags.asJava)
+  metricsGroup.newGauge(ReplicasToDeleteCountMetricName, () => replicasToDeleteCount, metricTags.asJava)
+  metricsGroup.newGauge(TopicsIneligibleToDeleteCountMetricName, () => ineligibleTopicsToDeleteCount, metricTags.asJava)
+  metricsGroup.newGauge(ReplicasIneligibleToDeleteCountMetricName, () => ineligibleReplicasToDeleteCount, metricTags.asJava)
+  metricsGroup.newGauge(ActiveBrokerCountMetricName, () => activeBrokerCount, metricTags.asJava)
   // FencedBrokerCount metric is always 0 in the ZK controller.
-  metricsGroup.newGauge(FencedBrokerCountMetricName, () => 0)
+  metricsGroup.newGauge(FencedBrokerCountMetricName, () => 0, metricTags.asJava)
```

Comment:
This new `metricsTags.asJava` argument is always an empty map in non-test paths, which is also the default used by the method overload without the new argument. So this causes no effective changes the controller metrics.